### ISSUE TITLE
delete adding gameCoins on frontend

### ISSUE THREAD

**Issue by leolabdev** [Alt-Org/Altzone-WebPages#144]
check https://github.com/Alt-Org/Altzone-Server/pull/238

as I remeber we had to hardcode it ?

**Comment by Skoivumaki** [comment]
Yes hardcoded kindof: On clan creation frontend gave API gameCoin value of 1 (was default).

gameCoins: yup
        .number()
        **.default(1)**
        .min(1, "Pelikolikoiden on oltava vähintään 1")
        .max(5000, "Pelikolikoiden on oltava enintään 1500"),

**Comment by leolabdev** [comment]
@MikhailDeriabin Will this update be deployed soon?

**Comment by leolabdev** [comment]
@Skoivumaki Mikhail announced an API update today, I think it can be removed now?

**Comment by Skoivumaki** [comment]
issue put on hold until #256 is complete.

**Comment by Skoivumaki** [comment]
![image](https://github.com/user-attachments/assets/42e26a3d-1df3-444e-8a10-4f0f6d5191ec)
Clan creation no longer need gameCoins to be declared.

**Comment by Skoivumaki** [comment]
Created new issue #398 that completes this.
